frontend/packages/dev-console/integration-tests/views/serverless.view.ts

```diff
@@ -0,0 +1,27 @@
+import { ExpectedConditions as by, element } from 'protractor';
+import { switchPerspective, Perspective } from './dev-perspective.view';
+
+export const firstKebabMenu = element.all(by.css('[data-test-id="kebab-button"]')).first();
+export const actionItems = element(by.css('[data-test-id="action-items"]'));
+export const editCount = element.all(by.css('.pf-c-dropdown__menu-item')).first();
+export const machineSetsInput = element(
+  by.className('pf-c-form-control co-m-number-spinner__input'),
+);
+export const submitCount = element(by.id('confirm-action'));
+export const cellClass = element.all(by.css('[data-test-id="openshift-machine-api"]')).first();
+export const pageSidebar = element(by.id('page-sidebar'));
+export const installServerlessOperator = element(
+  by.css('div.pf-c-modal-box.co-catalog-page__overlay.co-catalog-page__overlay--right'),
+);
+export const createProject = element(by.id('yaml-create'));
+export const nameofProject = element(by.id('input-name'));
+export const displayNameofProject = element(by.id('input-display-name'));
+export const servicesValidation = element(by.id('yaml-create'));
+export const serverlessNavItemValidation = element(by.css('[data-test-id="resource-title"]'));
+export const knativeServingNS = element(by.css('[data-test-id="knative-serving"]'));
+
+// Switch Perspective
+export const switchPersepctive = async () => {
+  await switchPerspective(Perspective.Developer);
+  await switchPerspective(Perspective.Administrator);
```

Comment:
What if I want to be on developer perspective after execution of this function, then will not work

Reply:
I have changed the name of the function to switchPerspectiveToAdmin so that it won't make any confusion.